### PR TITLE
You can now order gas canisters from cargo

### PR DESCRIPTION
**Pull request by DGamerL**
## What Does This PR Do
Adds the ability to order gas canisters with cargo, in all the normally accessible flavours! Plasma, CO2 and N2O are all  very high in demand by traitors and crew alike, so the prices of those are of course raised.

This also adds a destroy() to large crates.

## Why It's Good For The Game
In case of someone destroying the counter and all of the atmos air production gets nuked, this allows for small restocks so the station won't die a slow choking death.

## Testing
Went to cargo, ordered some canisters of Oxygen and Plasma. 
Used my god-given right of the second amendement to open the crates up.
Watched the world burn.

## Changelog
:cl:
add: adds the ability to buy gas canisters from cargo
tweak: large crates drop their contents when they are destroyed
/:cl:

### CODE REVIEW

File: code/game/objects/structures/crates_lockers/largecrate.dm [hunk 1]
```diff
@@ -49,6 +49,12 @@
 	else
 		return ..()
 
+/obj/structure/largecrate/Destroy()
+	for (var/obj/O in contents)
+		var/atom/movable/A = O
+		A.forceMove(get_turf(src))
```
Comment (Sirryan2002) [inline code/game/objects/structures/crates_lockers/largecrate.dm:55]:
```suggestion
	var/src_turf = get_turf(src)
	for(var/obj/O in contents)
		O.forceMove(src_turf )
```

File: code/modules/supply/supply_packs/pack_engineering.dm [hunk 2]
```diff
@@ -220,3 +220,45 @@
...
 	access = ACCESS_CE
 	announce_beacons = list("Engineering" = list("Chief Engineer's Desk", "Atmospherics"))
+
+/datum/supply_packs/engineering/canister/nitrogen
+	name = "Nitrogen canister"
+	contains = list(/obj/machinery/atmospherics/portable/canister/nitrogen)
+	cost = 50
+	containertype = /obj/structure/largecrate
+	containername = "nitrogen canister crate"
+
```
Comment (Sirryan2002) [inline code/modules/supply/supply_packs/pack_engineering.dm:230]: also you should probably make sure these are department restricted to engineering if they're not already

Comment (DGamerL) [reply code/modules/supply/supply_packs/pack_engineering.dm:230]: As far as I'm aware, it being a subtype of /supply_packs/engineering/ already puts it into the department restriction, but I could be very wrong here

**Review by Sirryan2002 — APPROVED** [review]

**Review by S34NW — APPROVED** [review]

File: code/game/objects/structures/crates_lockers/largecrate.dm [hunk 3]
```diff
@@ -49,6 +49,12 @@
 	else
 		return ..()
 
+/obj/structure/largecrate/Destroy()
+	var/src_turf = get_turf(src)
+	for(var/obj/O in contents)
+		O.forceMove(src_turf )
+	return ..()
```
Comment (S34NW) [inline code/game/objects/structures/crates_lockers/largecrate.dm:56]:
```suggestion
	for(var/obj/O in contents)
		O.forceMove(get_turf(src))
	return ..()
```

no point in declaring that earlier

**Review by S34NW — CHANGES_REQUESTED** [review]
Actually, one thing first

**Review by S34NW — APPROVED** [review]

**Pull request merged**